### DISCUSSION TITLE
Lua Scripting language

### DISCUSSION THREAD

**Discussion by stefaniecg** [pulsar-edit/.github#278] [Community Packages]
Is there a package for color highlighting LUA scripting language [1]?
if not, how can i make one?

Ref.
[1] https://www.lua.org

**Comment by confused-Techie** [comment]
Taking a look on all of our available community packages, there seems to be quite a few that offer grammar support for `.lua`, I haven't personally testing any of them so can't speak to their quality, but I'd hope at least one is able to provide you with what you are looking for.

But feel free to take a look at what's [available](https://web.pulsar-edit.dev/packages?fileExtension=lua).

Otherwise if you ever did want to try your hand at creating a community package that provides syntax highlighting our [documentation](https://docs.pulsar-edit.dev/developing-for-pulsar/creating-a-grammar/) attempts to cover it quite well.

**Reply by stefaniecg** [reply]
thank you for the link !
